### PR TITLE
Update build steps to allow specify namespace

### PR DESCRIPTION
**Pull request by Allda**
Namespace can be configured using pipeline parameter.

JIRA: ISV-855

### CODE REVIEW

File: pipelines/operator-ci-pipeline.yml [hunk 1]
```diff
@@ -207,14 +203,16 @@ spec:
           value: "$(tasks.preflight.results.log_output_file)"
         - name: result_file
           value: "$(tasks.preflight.results.result_output_file)"
+        - name: md5sum
```
Comment (MarcinGinszt) [inline pipelines/operator-ci-pipeline.yml:206]: Could you also add that to the hosted pipeline?

File: pipelines/operator-hosted-pipeline.yml [hunk 2]
```diff
@@ -318,6 +312,8 @@ spec:
           value: "$(tasks.operator-validation.results.package_name)"
         - name: preflight_results_exists
           value: "$(tasks.get-ci-results-attempt.results.preflight_results_exists)"
+        - name: md5sum
```
Comment (MarcinGinszt) [inline pipelines/operator-hosted-pipeline.yml:315]: This task doesn't have such parameter

**Comment by MarcinGinszt** [comment]
LGTM- after fixing comments

File: pipelines/operator-hosted-pipeline.yml (hunk 2 above)
Comment (Allda) [reply pipelines/operator-hosted-pipeline.yml]: You're right. I put a parameter to a wrong task.

File: pipelines/operator-ci-pipeline.yml (hunk 1 above)
Comment (Allda) [reply pipelines/operator-ci-pipeline.yml:206]: Done

**Comment by MarcinGinszt** [comment]
LGTM!

**Pull request merged**